README.md

```diff
@@ -69,83 +69,109 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.55 - January 2022 Update
+### 0.56 - February 2022 Update
 
-In this release, we are continuing our progress toward getting PowerToys ARM64 ready, fix some top issues and new utilities.  Work from last month enabled us to upgrade the code base to .NET 5 and next month onward to .NET 6. This will provide stability and speed improvements.
+In this release, we focused heavily on stability and improvements.  Below are some of the highlights!
 
-We're also extremely excited to bring on 3 new PowerToy utilities.
+**Highlights**
 
-- File Explorer add-on: Developer files for preview pane. This should add about 150 file extensions total. We are using the [Monaco Editor](https://github.com/Microsoft/monaco-editor) to power this experience.  Thanks [@aaron-junker](https://github.com/aaron-junker)!
-- File Explorer add-on: STL file format thumbnail generation!  Since STL is a common 3D file format, this allows a quick visual check. Thanks [@pedrolamas](https://github.com/pedrolamas)!  Preview pane support is already in Windows.
-- Mouse Utility: Crosshair over pointer via <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>P</kbd>. This feature was co-developed with the accessibility team at Microsoft. When the team told us about the idea and described trying to find your cursor by looking through a straw, we knew we could leverage code from the other mouse utilities to quickly enable this feature. Below is a quote from one of the testers with a rough validation build:
+- Upgrade notes!  A big feedback items for new versions was upgrade notes.  We have the release notes on GitHub as well.
+- FancyZones zone limit has been increased to 128 zones.  Before the limit was 40.
+- Timezone conversion plugin for PowerToys Run!  Thanks [@TobiasSekan](https://github.com/TobiasSekan)
+- Child and Popup window support for FancyZones.  To enable, go to Settings→FancyZones→Windows. These were bugs we fixed that without realizing it was a useful feature to most.
+- Find my mouse will now activate via shaking the mouse with a settings change!
+![Find my mouse setting for Activate to shake](https://user-images.githubusercontent.com/26118718/153509687-e99cc9ce-28bf-481b-b27b-39715cb3d82a.png)
```

Comment:
outdated screenshot. setting moved between enable switch and expander. Updated screenshot should existing on the pr discussion.